joplin/locations/models.py

```diff
@@ -126,55 +160,29 @@ class LocationPage(JanisBasePage):
     ]
 
 
-class LocationPageRelatedServices(ClusterableModel):
+class LocationPageRelatedServices(Orderable):
```

Comment:
Figured content might want to actually change the order in which related services appear